docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -346,91 +315,108 @@ bundling solution people are using.
 
-**CssModulesButton.css**
+**CssModulesSlider.module.css**
 
 ```css
-.button {
-  background-color: #6772e5;
-  color: #fff;
-  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-  padding: 7px 14px;
+.slider {
+  color: #20b2aa;
 }
-.button:hover {
-  background-color: #5469d4;
+.slider:hover {
+  color: #2e8b57;
 }
 ```
 
 **CssModulesButton.js**
 
 ```jsx
-import * as React from 'react';
+import React from 'react';
 // webpack, parcel or else will inject the CSS into the page
-import styles from './CssModulesButton.css';
-import Button from '@material-ui/core/Button';
+import styles from './CssModulesSlider.module.css';
+import Slider from '@material-ui/lab/SliderStyled';
 
-export default function CssModulesButton() {
+export default function PlainCSSSlider() {
   return (
     <div>
-      <Button>Default</Button>
-      <Button className={styles.button}>Customized</Button>
+      <Slider defaultValue={30} />
+      <Slider className={styles.slider} defaultValue={30} />
     </div>
   );
 }
 ```
 
 ### Controlling priority ⚠️
 
-**Note:** JSS injects its styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/styles/advanced/#css-injection-order), as in the demo:
+**Note:** Most of the CSS-in-JS solutions inject their styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change the CSS injection order, as in the demo:
 
 ```jsx
-import { StylesProvider } from '@material-ui/core/styles';
+import * as React from 'react';
+import { CacheProvider } from '@emotion/core';
+import createCache from '@emotion/cache';
+
+const head = document.getElementsByTagName('head')[0];
 
-<StylesProvider injectFirst>{/* Your component tree.
-      Now, you can override Material-UI's styles. */}</StylesProvider>;
+const emotionContainer = head.insertBefore(
+  document.createElement('STYLE'),
+  head.firstChild,
+);
+
+const cache = createCache({
+  container: emotionContainer,
+});
+
+export default function App() {
+  return (
+    <CacheProvider value={cache}>
+      {/* Your component tree. Now you can override Material-UI's styles. */}
+    </CacheProvider>
+  );
+}
 ```
 
 ### Deeper elements
 
-If you attempt to style a Drawer with variant permanent,
-you will likely need to affect the Drawer's child paper element.
-However, the paper is not the root element of Drawer and therefore styled-components customization as above will not work.
-You need to use the [`classes`](/styles/advanced/#overriding-styles-classes-prop) API of Material-UI.
+If you attempt to style the Slider,
+you will likely need to affect some of the Slider's child elements, for example the thumb.
+However, the thumb is not the root element of Slider and therefore customization as above will not work.
+You need to use the `componentProps` API of Material-UI, in order to provide custom classNames for the slots.
```

Comment:
doesn't match the example, I think that `componentProps` is only good for the Plain CSS example

Reply:
I've left the `componentsProps` only in the Plain CSS and CSS modules examples...